src/app/public/modules/repeater/repeater-item.component.ts

```diff
@@ -526,7 +526,8 @@ export class SkyRepeaterItemComponent implements OnDestroy, OnInit, AfterViewIni
 
   private toggleSelected(): void {
     if (this.selectable) {
-      this.isSelected = !this.isSelected;
+      this._isSelected = !this._isSelected;
+      this.isSelectedChange.emit(this._isSelected);
```

Comment:
I see we're essentially duplicating the logic from the `updateIsSelected` method. What if we consolidated the change detection and put it in the `isSelected` setter? @Blackbaud-AlexKingman Any red flags going off with this suggestion?
```
@Input()
public set isSelected(value: boolean) {
  if (value !== this._isSelected) {
    this._isSelected = value;
    this.isSelectedChange.emit(this._isSelected);
  }
}
```

Reply:
I thought about that initially.  Wasn't sure how the two-way binding behaves in that case (hopefully doesn't loop).  I can try it out.  Worst case, just don't emit if the value hasn't changed.

Reply:
@Blackbaud-SteveBrush I tested your code from above to place the changes in the public setter, and only emit when the value has changed.  It works well.  So I can go ahead and make that change if @Blackbaud-AlexKingman agrees it's a good approach.

Reply:
Thanks @michael-tims. Yes, I'd also prefer that being isolated to the setter.

Reply:
Done